debt/views.py

```diff
@@ -25,9 +25,10 @@ def post(self, request):
         )
         # if the user is authenticated save to account and take to overview
         if request.user.is_authenticated():
-            render(request, 'overview.html')
+            return redirect('/overview')
 
-        template_vars = {
-            'get_started': 'Save!',
-        }
-        return render(request, 'sign-in.html', template_vars)
+        else:
+            template_vars = {
+                'get_started': 'Save!',
+            }
+            return redirect('/sign-in', template_vars)
```

Comment:
is the `/`  required? can you just `return redirect('sign-in', template_vars)`?

Even better solution: (if it works)
```
from account.urls import SIGN_IN
...
return redirect(SIGN_IN, template_vars)
```